presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -1055,4 +1067,14 @@ public static boolean isVerboseRuntimeStatsEnabled(ConnectorSession session)
     {
         return session.getProperty(VERBOSE_RUNTIME_STATS_ENABLED, Boolean.class);
     }
+
+    public static boolean getDwrfWriterStripeCacheEnabled(ConnectorSession session)
```

Comment:
we use `is` for boolean functions. So may be rename to `isDwrfWriterStripeCacheEnabled`

So may be also change `getDwrfStripeCacheEnabled`-> `isDwrfStripeCacheEnabled` in `OrcFileWriterConfig.java` as well